tasks/update_go.py

```diff
@@ -31,7 +31,7 @@ def update_go(
     """
     import semver
 
-    if not semver.Version.is_valid(version):
+    if not semver.VersionInfo.isvalid(version):
```

Comment:
Out of curiosity, is this (and the similar one below) change related to the go update itself?

Reply:
I created the task when I did the last Go update, but it seems like I was using the wrong version of `semver` locally...